graph/options.go

```diff
@@ -84,6 +84,20 @@ type RequestedAppenders struct {
 	AppenderNames []string
 }
 
+func (ra *RequestedAppenders) Includes(name string) bool {
```

Comment:
Is this utility still used after the impl change?  If not it can be removed,

Reply:
Nope it can be removed